MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -8,29 +8,129 @@
  * [Switch Statements](https://github.com/learn-co-curriculum/swift-switchStatement-readme)
 
  */
-/*: question1
- ### 1. Question involving switches. La di da.
- */
+
+
+// First let's try some of last lab's questions but using a 'switch' instead of 'if' or 'if/else'.
+
+// Given....
+let x = 5.0
+let y = 12
+let a = 321
+let b = 32
+
+
+/*: Question 1
+### 1. Print "true" if a modulo b is zero
```

Comment:
Similar change to the prior lab. Adding the "is equal to zero part" so it reads as.

Print "true" if a modulo b is equal to zero.